src/main/java/GulieUi.java

```diff
@@ -0,0 +1,81 @@
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.io.PrintStream;
+import java.time.format.DateTimeFormatter;
+import java.util.Scanner;
+
+public class GulieUi {
+    private static final String name = "Güliedistodiez";
+    private static final String line = "____________________________________________________________";
```

Comment:
Remember to change these constants too!

```suggestion
    private static final String NAME = "Güliedistodiez";
    private static final String LINE = "____________________________________________________________";
```